c10/util/env.h

```diff
@@ -1,41 +1,23 @@
 #pragma once
 
-#include <c10/util/Exception.h>
-#include <cstdlib>
-#include <cstring>
+#include <c10/macros/Export.h>
 #include <optional>
+#include <string>
 
 namespace c10::utils {
+
+// Checks an environment variable is set.
+C10_API bool has_env(const char* name);
+
+// Reads an environment variable and returns the content if it is set.
+C10_API std::optional<std::string> get_env(const char* name);
```

Comment:
Should we document that we discourage use of this method for boolean checks in favor of check_env below to improve consistency of handling?

Reply:
Sometimes we want the specific contents of env variables for further string comparison, so they are used for different purposes.

Reply:
Ho yes, that why I'm saying that this one should be used only for non-boolean env.

Reply:
I see what you mean, let's add a comment.